webextensions/api/management.json

```diff
@@ -363,10 +363,10 @@
                 "version_added": "79"
               },
               "firefox": {
-                "version_added": false
+                "version_added": "55"
               },
               "firefox_android": {
-                "version_added": false
+                "version_added": "55"
               },
```

Comment:
For onUninstalled we may want to point out in a note that the onUninstalled events is not emitted when the extension is in the "pending uninstall" state, once it is completely removed (e.g. when the about:addons tab is closed or refreshed) the onUninstalled event is emitted as expected.

Reply:
Thanks! I updated this to add a note :)